qmk/config.h

```diff
@@ -37,7 +37,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #define TAPPING_FORCE_HOLD_PER_KEY
 #define TAPPING_TERM_PER_KEY
 #define TAPPING_TERM 200
-#define IGNORE_MOD_TAP_INTERRUPT
```

Comment:
Why was this removed? Is there a replacement for `IGNORE_MOD_TAP_INTERRUPT` in the new version?